hexrd/ui/calibration/panel_buffer_dialog.py

```diff
@@ -160,3 +161,18 @@ def mode(self, v):
     def update_mode_tab(self):
         mode_tab = getattr(self.ui, self.mode + '_tab')
         self.ui.tab_widget.setCurrentWidget(mode_tab)
+
+    def clear_panel_buffer(self):
+        # Clear the config options on the internal config
+        config = HexrdConfig().config
+        detector_config = config['instrument']['detectors'][self.detector]
+
+        buffer_default = {'status': 0}
+        buffer = detector_config.setdefault('buffer', buffer_default)
+
+        # Must match the detector size
+        detector_shape = (detector_config['pixels']['rows']['value'],
+                          detector_config['pixels']['columns']['value'])
+
+        empty_mask = np.ones(detector_shape, dtype=bool)
+        buffer['value'] = empty_mask
```

Comment:
I believe that we can just set `buffer['value'] = [0., 0.]` to nullify it, or even `buffer['value'] = None` (but we would need to make sure we are handling the `None` case properly in the GUI - I think we are not, even though it is valid in hexrd).

Reply:
Much cleaner, thank you. Fixed to use the `buffer['value'] = [0., 0.]` option.